app/build.gradle

```
@@ -36,13 +39,14 @@ android {
         viewBinding true
     }
 
-buildFeatures {
-   dataBinding true
-}
+    buildFeatures {
+       dataBinding true
+    }
 }
 
 dependencies {
-    implementation 'com.github.bumptech.glide:glide:4.8.0'
+    implementation 'com.github.bumptech.glide:glide:4.12.0'
+    annotationProcessor 'com.github.bumptech.glide:compiler:4.12.0'
```

사실 이건 내가 세미나때 조금 잘못 알려준 부분이 있는데, 코틀린의 경우 **kapt**를 이용해서 어노테이션 프로세싱을 진행하기 때문에

```suggestion
    kapt 'com.github.bumptech.glide:compiler:4.12.0'
```
이렇게 dependencies에 추가해주는 게 좋을것같아요 :)

---

넵~! 수정했습니다😊